physics/test/IceGrowth_test.cpp

```diff
@@ -126,7 +126,7 @@ TEST_CASE("New ice formation", "[IceGrowth]")
     ocnBdy.updateBefore(tst);
     ig.update(tst);
 
-    ModelArrayRef<ModelComponent::SharedArray::NEW_ICE, MARBackingStore, RO> newice(ModelComponent::getSharedArray());
+    ModelArrayRef<Shared::NEW_ICE, RO> newice(ModelComponent::getStore());
 
     double prec = 1e-5;
     REQUIRE(newice[0] == Approx(0.0258264).epsilon(prec));
```

Comment:
This looks like it might be a good test (and imo this line change summarises the PR)  but needs a stated test case objective and comment for each potential test point (or a failure comment ---- I think catch let's that happen, can't remember)